src/visbook/visbook.api.ts

```diff
@@ -0,0 +1,60 @@
+import { AccommodationAvailability } from './visbook.interface';
+
+export class VisbookApi {
+  async isCabinAvailable(
+    cabinVisbookId: number,
+    checkIn: string,
+    checkOut: string,
+  ): Promise<boolean> {
+    const visbookResponse = await this.makeRequest(
+      cabinVisbookId,
+      checkIn,
+      checkOut,
+    );
+
+    const accommodations = visbookResponse.accommodations;
+
+    if (accommodations === undefined) {
+      return false;
+    }
+
+    for (const accommodation of accommodations) {
+      if (accommodation.availability.available === true) return true;
+    }
+
+    return false;
+  }
```

Comment:
Perhaps it would be better to return the number of beds that are available? Then we could say something like "10/20 beds are available"